rep-0003.rst

```diff
@@ -220,18 +216,64 @@ Exact or Series Requirements:
 - Qt 5.3.x
 - PyQt5
 
-Buildsystem support:
+Build System Support:
 
-- catkin:
+- Same as Indigo
 
-  - build from source
-  - release for binary packaging
-  - wiki documentation
-  - continuous integration
+Lunar Loggerhead (May 2017 - May 2019)
+--------------------------------------
+Required Support for:
 
-- rosbuild:
+- Ubuntu Xenial (16.04)
+- Ubuntu Yakkety (16.10)
+- Ubuntu Zesty (17.04)
 
-  - build from source
+Recommended Support for:
+
+- Debian Stretch
+- Fedora 26
+
+Architectures Supported:
+
+- amd64
+- arm32
+- arm64
```

Comment:
It was on this list before but we decided [to remove it](https://github.com/ros-infrastructure/rep/pull/123/commits/730c40995b031bc99402b503c0b871c13de0fb5f) in favor of an [additional paragraph below](https://github.com/ros-infrastructure/rep/pull/123/files#diff-41ae062315c36c45afaa5c192b2ca7c5R298). We could either:
- extend the paragraph to highlight that we still expect the code to compile against i386
- split this list into two (analog to the lists of platforms):
  - one for the architectures we provide binary packages for
  - one for the architectures we recommend people's code to build on.

Not sure if we can say required in this case, e.g. ARM support is not "required" given that we don't prevent maintainers to release packages that don't compile on these platforms. Additionally we can't enforce that it compiles on i386 if we don't build them on the buildfarm.